src/org/thoughtcrime/securesms/database/model/MessageRecord.java

```diff
@@ -86,6 +88,11 @@ public boolean isPinned() {
     return MmsSmsColumns.Types.isPinned(type);
   }
 
+  public boolean hasEmojiReaction() {
+    // To implement
```

Comment:
should this be implemented before merge?

Reply:
I would encourage adding `TODO` before comments of this type to make them easier to find.

Reply:
@g-harel it is used in conversationFragment and not sure if I can delete it @Dan-Yoo